backend/tests/unit/core/schema/schema_branch/test_validate_computed_attributes.py

```diff
@@ -373,3 +373,43 @@ async def test_schema_computed_attribute_violations(schema_root: SchemaRoot, exp
         schema.process()
 
     assert str(exc.value) == expected_error
+
+
+async def test_valid_datetime_computed_attribute():
```

Comment:
This test is failing, can you execute it with pytest to see the error message and fix the issue